packages/nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -46,7 +48,8 @@ export class RedirectsMiddleware extends MiddlewareBase {
     super(config);
     // NOTE: we provide native fetch for compatibility on Next.js Edge Runtime
     // (underlying default 'cross-fetch' is not currently compatible: https://github.com/lquixada/cross-fetch/issues/78)
-    this.redirectsService = new GraphQLRedirectsService({ ...config, fetch: fetch });
+    this.redirectsService =
```

Comment:
The same as for client should we put these services overrides under some subproperty?